lib/mindwendel/permissions.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Mindwendel.Permissions do
+  @moduledoc """
+  The Permissions context.
+  """
+  def has_moderating_permission(brainstorming, current_user) when current_user != nil do
+    Enum.member?(current_user.moderated_brainstormings |> Enum.map(& &1.id), brainstorming.id)
```

Comment:
I wrote a small essay on this, but this is the "single pipe" which is discouraged: https://github.com/rrrene/credo/issues/221